src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/KafkaMessageConsumer.cs

```diff
@@ -30,14 +31,18 @@ public KafkaMessageConsumer(ReceiveSettings receiveSettings, KafkaReceiveEndpoin
         {
             _receiveSettings = receiveSettings;
             _context = context;
+            _limit = new SemaphoreSlim(receiveSettings.ConcurrentMessageLimit);
```

Comment:
Is this ConcurrentMessageLimit or PrefetchCount? In Event Hub, you used prefetch count...

Reply:
yeah, I missed that